packages/next/src/lib/metadata/types/twitter-types.ts

```diff
@@ -64,16 +64,16 @@ type TwitterPlayerDescriptor = {
 }
 
 type ResolvedTwitterImage = {
-  url: null | URL
+  url: null | URL | string
   alt?: string
 }
 type ResolvedTwitterSummary = {
-  site: string | null
-  siteId: string | null
-  creator: string | null
-  creatorId: string | null
-  description: string | null
-  title: AbsoluteTemplateString
+  site?: string | null
+  siteId?: string | null
+  creator?: string | null
+  creatorId?: string | null
+  description?: string | null
```

Comment:
I think we shouldn't allow `undefined` for resolved type, since we got `null` already?

Reply:
reverted